tests/test_decorators.py

```diff
@@ -42,4 +42,5 @@ def f ():
             time.sleep(1)
             return 42
         self.assertEqual(f(), 42)
-        self.assertTrue(log.getvalue())
+        log_value = log.getvalue()
+        self.assertEquals(log_value, b'f took 1.00 seconds\n()\n{}\n')
```

Comment:
assertEqual**s** emits a deprecation warning; please use `assertEqual`.

(And, again, I don't understand why this needs to be bytes?  Especially given that it's writing to a StringIO and not a BytesIO?)

Reply:
It was using:
`log = BytesIO()`
Changed to StringIO in python3_05c.